packages/api-server/src/methods/modules/eth.ts

```diff
@@ -39,6 +39,9 @@ const SUDT_PAY_FEE_LOG_FLAG = "0x1";
 const POLYJUICE_SYSTEM_LOG_FLAG = "0x2";
 const POLYJUICE_USER_LOG_FLAG = "0x3";
 
+const HEADER_NOT_FOUND_ERR_CODE = -32000;
```

Comment:
suggest moving to `packages/api-server/src/methods/error-code.ts`

Reply:
updated